src/applications/gi/components/content/StemScholarshipNotification.jsx

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
+import environment from '../../../../platform/utilities/environment';
```

Comment:
Relative paths work for imports, so this could just be `'platform/utilities/environment'`.

Reply:
Thanks for the tip U-DON